crates/python/tests/qpu/test_api.py

```diff
@@ -44,3 +47,15 @@ def test_submit_retrieve(
 
     job_id = submit(program, memory, quantum_processor_id)
     results = retrieve_results(job_id)
+
+class TestPickle():
+    @pytest.mark.parametrize("strategy", [ConnectionStrategy.gateway(), ConnectionStrategy.direct_access(), ConnectionStrategy.endpoint_id("endpoint_id")])
+    def test_connection_strategy(self, strategy: ConnectionStrategy):
+        pickled = pickle.dumps(strategy)
+        unpickled = pickle.loads(pickled)
```

Comment:
Unless I'm missing something about pytest, this test is missing an `assert`

Reply:
Oops, good catch